scripts/ci/check_compliance.py

```diff
@@ -30,8 +30,6 @@
     # Propagate this decision to child processes.
     os.environ['ZEPHYR_BASE'] = ZEPHYR_BASE
 
-    print(f'ZEPHYR_BASE unset, using "{ZEPHYR_BASE}"')
```

Comment:
we should in fact automatically guess Zephyr base, script is at $ZEPHYR_BASE/ci

Reply:
We are doing exactly that a few lines above.